utils/__init__.py

```diff
@@ -38,3 +40,33 @@ def generate_heroku_app_name(deployment_name):
     invalid_chars = re.compile("[^a-zA-Z0-9-]")
     app_name = re.sub(invalid_chars, "-", app_name)
     return app_name.lower()
+
+
+def validate_configuration_value(configuration):
+    schema = {
+    "dyno_counts": {
+        "type": "integer",
+        "required": True,
+        "min": 1
+    },
+    "dyno_type": {
+        "type": "string",
+        "allowed": [
+            "free",
+            "hobby",
+            "standard-1x",
+            "standard-2x",
+            "performance-m",
+            "performance-l"
+        ],
+        "required": True
+        },
+    }
+
+    v = cerberus.Validator(schema)
+    if v.validate(configuration, schema):
```

Comment:
also just a bit neat:
```python
class ValidationError(Exception):...

if not v.validate(configuration):
	raise ValidationError(...)
```

Reply:
Thanks! I'll update it asap tomorrow (doing hw rn)